### DISCUSSION TITLE
Versioned Doks

### DISCUSSION THREAD

**Discussion by james-d-elliott** [thuliteio/doks#432] [Q&A] [answered]
Hey I'm just wondering if there are any example projects anyone knows of that have a versioned layout?

i.e. for version 1.1.x of a product one set of docs, and for 1.2.x another set of docs? Particularly interested in if there is a drop down etc?

**Comment by h-enk** [comment, marked as answer]
Hey @james-d-elliott, it's in the making and will be part of the upcoming Doks release (available soon).

Here's a POC (still very alpha — only the 0.3 + 0.1 trees are working):

- [/docs/0.3/prologue/introduction/](https://doks-versioning-poc.netlify.app/docs/0.3/prologue/introduction/)
- [/docs/0.1/prologue/introduction/](https://doks-versioning-poc.netlify.app/docs/0.1/prologue/introduction/)

And here's some background (internal):
- https://doks-versioning-poc.netlify.app/docs/0.3/extensions/versioning/

**Comment by james-d-elliott** [comment]
Nice! I'll aim to work towards testing this over the coming weekend. :) Also github is not letting me mark as answer.
